src/types/organization/index.ts

```
@@ -1,51 +1,38 @@
-  ROLE_OWNER,
-  ROLE_VIEWER,
-  ROLE_COMMENTER,
-  ROLE_EDITOR,
-] as const
-
-export const UserTypes = [ROLE_MEMBER, ROLE_SUPER_ADMIN, ROLE_USER] as const
-
-export type MemberRole = (typeof MemberRoles)[number]
-export type UserType = (typeof UserTypes)[number]
+import type { Zone } from "../zones"
+import { UserRole} from "../user"
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)